server/views/actions.njk

```diff
@@ -5,12 +5,24 @@
     {% from "govuk/components/radios/macro.njk" import govukRadios %}
     {% from "govuk/components/button/macro.njk" import govukButton %}
     {% from "govuk/components/error-message/macro.njk" import govukErrorMessage %}
-
+    {% if model.items.length > 0 %}
```

Comment:
I meant to include these view changes in the parcel 'description' PR, but forgot to push before it was reviewed and merged... be good to get them in as part of this PR.